src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -233,6 +233,19 @@ public void LogKustoTokenRefreshSummary(string Source, string Message, long Late
                 ExceptionDetails);
         }
 
+        [Event(3004, Level = EventLevel.Informational, Channel = EventChannel.Admin, Message = ETWMessageTemplates.LogKustoQueryInformation)]
+        public void LogKustoQueryInformation(string Source, string RequestId, string Message, long LatencyInMilliseconds, string Details, string Content, string ExceptionType, string ExceptionDetails)
```

Comment:
QQ : will this add a new column for Content in Kusto or is it already there

Reply:
It is already there